### DISCUSSION TITLE
Always on top feature

### DISCUSSION THREAD

**Discussion by SilverReceiver** [sylikc/jpegview#190] [Ideas]
I wanted the application to be able to be always on top so I looked into the code and made the necessary changes and build the application and it worked, all I did was change a few parameters on SetWindowPos in MainDlg.cpp on line 317, from HWND_TOP to HWND_TOPMOST and removing SWP_NOZORDER.

So like this,
::SetWindowPos(m_hWnd, HWND_TOPMOST, m_monitorRect.left, m_monitorRect.top, m_monitorRect.Width(), m_monitorRect.Height(), SWP_NOCOPYBITS | SWP_NOACTIVATE | SWP_NOREDRAW);

The thing is I don't know if this breaks anything else and I would love to hear what other people with expertise think about this and maybe make it a setting you can toggle in config. I posted about this in discussions on the Sourceforge page but I realized that there is not much activity there so here I am.

**Comment by sylikc** [comment]
Always on top is already a feature on the latest version.

Shift+F12 is the default key map

**Comment by sylikc** [comment]
There's not yet the feature to start in always on top mode, it's planned though along with the borderless title bar mode... As it seems like there's enough interest

**Reply by SilverReceiver** [reply]
Cool, didn't realize that. Looking forward to that and borderless as well, thank you for replying.
